compiler/src/dmd/typesem.d

```diff
@@ -2631,6 +2631,31 @@ extern(C++) Type typeSemantic(Type type, const ref Loc loc, Scope* sc)
     }
 }
 
+extern(C++) Type trySemantic(Type type, const ref Loc loc, Scope* sc)
```

Comment:
this (now free) function is unused from C++ in LDC, does it need to be `extern(C++)` at all?

Reply:
It's easier for me to just keep the interface as is, instead of double checking every time I make a PR.

Reply:
OK, please do a follow up PR then.